mkosi/__init__.py

```diff
@@ -4848,6 +4848,23 @@ def parse_remove_files(value: str) -> List[str]:
     return ["/" + os.path.normpath(p).lstrip("/") for p in value.split(",") if p]
 
 
+def parse_ssh_agent(value: str) -> Union[Path, bool]:
+    """Can be True/False or a path to a socket."""
+
+    try:
+        if not parse_boolean(value):
+            return False
+    except ValueError:
+        return Path(value)
+
+    sock = os.getenv("SSH_AUTH_SOCK")
+    if sock is None:
+        die(
+            "--ssh-agent=true but cannot parse $SSH_AUTH_SOCK (consider running 'sudo' with '-E')"
+        )  # NOQA: E501
```

Comment:
Can we please write this as `die("message")` ? This black misformatting is madness.

Reply:
sure, just cargo-culted it over